### PR TITLE
tools/labs: Add TODO for exercise 7-list-test

### PR DESCRIPTION
**Pull request by teoserbanescu**
Modify the TODOs in list-sync as follows: TODO 1 to solve exercise 6-list-sync and TODO 2 to solve exercise 7-test-list.

### CODE REVIEW

**Comment by valx90** [comment]
We should also change the exercise text to specify which TODO must be filled.

**Comment by teoserbanescu** [comment]
I agree. Should this be done here or in another commit referring to the documentation("Documentation:teaching:labs:kernel_api")?

File: tools/labs/templates/kernel_api/6-list-sync/list-sync.c [hunk 1]
```diff
@@ -24,7 +24,7 @@ struct task_info {
 
 static struct list_head head;
 
-/* TODO: you can use either a spinlock or rwlock, define it here */
+/* TODO 1: you can use either a spinlock or rwlock, define it here */
 DEFINE_RWLOCK(lock);
```
Comment (dbaluta) [inline tools/labs/templates/kernel_api/6-list-sync/list-sync.c]: @teoserbanescu careful here. Please also check that the output skeleton looks fine. 

AFAIK a syntax like:
```
TODO 1/3

foo1();
foo2();
foo3();
```

removes first 3 lines after TODO from output skeleton.

I'm not sure what ```TODO 1``` does. Maybe is as expected. But please check the output.

Comment (teoserbanescu) [reply tools/labs/templates/kernel_api/6-list-sync/list-sync.c]: It was behaving as expected, removing only one line. I still modified this in another commit to be more clear.

**Comment by valx90** [comment]
@teoserbanescu we shouldn't have any merge commits on the source branch, because they will also end up in master.
I suggest doing rebase instead of merge, to prevent those commits.

For now, we can squash all the commits in a single one.

File: tools/labs/templates/kernel_api/6-list-sync/list-sync.c (hunk 1 above)
Comment (valx90) [reply tools/labs/templates/kernel_api/6-list-sync/list-sync.c]: @dbaluta the number of lines is 1 by default if you skip it.

**Pull request merged**